spec/factories/courses.rb

```diff
@@ -93,6 +93,7 @@
 
     trait :with_accrediting_provider do
       association(:accrediting_provider, factory: :provider)
+      accrediting_provider_code { accrediting_provider.provider_code }
```

Comment:
Why is this needed? `accrediting_provider` is a `belongs_to`, although a fairly complex one, but I just tested this locally and it seems to work as expected without this change.

Reply:
Yea that was left over a provider_id wasn't being assigned and my previous code was reliant on it. It should have just been removed though not changed to use the provider code. Have now taken in out.